src/api/prices/prices.module.ts

```diff
@@ -16,7 +17,7 @@ import { PricesService } from './prices.service';
     TypeOrmModule.forFeature([Game, Price, Token]),
```

Comment:
import GamesModule & TokensModule Instead

Reply:
Marked as Resolved due to outdated.